.golangci.yaml

```diff
@@ -28,21 +28,25 @@ linters:
     - wastedassign
     - whitespace
     - protogetter
-    # TODO enable https://github.com/openfga/openfga/issues/598
-    # - bodyclose
-    # - contextcheck
-    # - exhaustive
-    # - errorlint
-    # - gochecknoglobals
-    # - gocritic
-    # - godot
-    # - gosec
-    # - noctx
-    # - inamedparam
-    # - perfsprint
-    # - prealloc
-    # - tparallel
+    - revive
+    - godot
+    - bodyclose
+    - gocritic
+    - godot
+    - gocyclo
+    #- gosec Too many different issues. Needs to be tackled in a separate PR.
```

Comment:
This is important to have as a linter within the config as it gives us insights into potential security issues, however due to the big amount of issues reported this needs to be tackled as a separate and dedicated PR.

Reply:
what about `contextcheck`, `exhaustive`, `errorlint`, `gochecknoglobals`, `noctx`, `inamedparam`, `persprint`, `prealloc` and `tparallel`? can we keep them as comments? or did you see no value in them?

Reply:
I honestly felt that they hinder more than aid at this point in time, as they flagged too many issues that aren't really problems. For example we make quite heavy use of global vars through out the codebase and having `gochecknoglobals` would not bring much of a benefit at this point in time. I think it would be kind of frustrating to get bogged down by minor things when you’re in the middle of solving bigger problems. Once we bring the codebase in a better shape we could consider them again, I'd rather not leave them commented out but if you feel strongly about this I can add comments for them, let me know.